runners/core-java/src/test/java/org/apache/beam/runners/core/metrics/MetricsPusherTest.java

```diff
@@ -73,7 +74,12 @@ public void pushesUserMetrics() throws Exception {
     assertThat(TestMetricsSink.getCounterValue(COUNTER_NAME), is(NUM_ELEMENTS));
   }
 
-  @Category({ValidatesRunner.class, UsesAttemptedMetrics.class, UsesCounterMetrics.class})
+  @Category({
+    ValidatesRunner.class,
+    UsesAttemptedMetrics.class,
+    UsesCounterMetrics.class,
+    UsesSystemMetrics.class
```

Comment:
Just to double check since I don't really know what all the categories mean - is it possible that `UsesMetricsPusher` is relevant?

Reply:
I don't think so - MetricsPusher is used in both portable and legacy modes of runners. 
(eg. in [FlinkRunner](https://github.com/apache/beam/blob/c2f0d282337f3ae0196a7717712396a5a41fdde1/runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkRunner.java#L137) )
The issue here is that legacy Flink runner doesn't expose system metrics, so the test involving them made the whole suite fail.